cirq-core/cirq/transformers/transformer_primitives.py

```diff
@@ -275,8 +286,16 @@ def apply_merge_func(op1: ops.Operation, op2: ops.Operation) -> Optional[ops.Ope
                     ).with_tags(*op.tags, _circuit_op_tag)
                 )
                 continue
+
+            op_ck = set(op.classical_controls)
+
+            def get_idx(op_qs: Set['cirq.Qid']):
+                op = ops.identity_each(*op_qs).with_classical_controls(*op_ck)
```

Comment:
`op` is already used - pick a new name to avoid confusion.

Reply:
done